apps/docs/hooks/Math/useAbs/src/abs.config.ts

```
@@ -0,0 +1,50 @@
+  COMPUTE = 'COMPUTE',      // Basic calculation of the absolute value
+  VALIDATE = 'VALIDATE',    // Input validation
+}
+
+/** Configuration for operations */
+interface OperationsConfig {
+  readonly core: readonly CoreOperations[];
+  readonly addons: readonly string[];
+}
+
+/** Options for absolute value calculations */
+interface AbsOptions {}
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)